discover/Home/getting-started/creating-account.mdx

```diff
@@ -0,0 +1,19 @@
+---
+title: "Creating a Sophon account"
+description: "An overview of Sophon Accounts and its unique benefits."
+---
+
+Sophon Account makes interacting with crypto as simple as using your favourite apps. It removes the typical web3 barriers while maintaining the security and ownership you deserve.
+
+Creating a Sophon Account is easy - just go to the “Login” button in the top right of any page.
+
+Behind the scenes, Sophon Account uses advanced account abstraction technology to create a secure, self-custodial wallet. This means:
+
+- **Simple sign-in**: Use familiar methods like Google, Email, Discord, Telegram or X
+- **No extensions needed**: Everything happens directly in the app you're using
+- **Self-custodial security**: You maintain full ownership of your assets and data
+- **Seamless transactions**: Approve actions directly within applications, without switching between windows
```

Comment:
The pop up can be considered a switch between windows.  Maybe let's remove this one only. And one for Modularity
**Modular**: Install and remove modules that fits your daily needs.
**Extra security**: Define guardians that can help you regain access in any scenario that led you to lose access to your account.